Esri/arcgis-maps-sdk-swift-samples
# [New] Apply dictionary renderer to feature layer

## Description

This PR implements `Apply dictionary renderer to feature layer` in `Visualization` category.
URL to README: [Apply dictionary renderer to feature layer](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/New-ApplyDictionaryRendererToFeatureLayer/Shared/Samples/Apply%20dictionary%20renderer%20to%20feature%20layer)

## Linked Issue(s)

- `swift/issues/6246`

## How To Test

- Open the sample to see the feature layers rendered with the dictionary renderer.

## Screenshots

![apply-dictionay-renderer-to-feature-layer](https://github.com/user-attachments/assets/ef04b52b-5f9e-4cce-877b-c4f01243e28a)
